scripts/pi-install.sh

```diff
@@ -160,11 +160,11 @@ cat << "EOF"
 
         |        The smart-mirror is now installed!
        / \       
-      / _ \      To configure audio input, speech, and the mirror itself, please visit:
+      / _ \      For configuration instructions, check out the documentation at:
      |.o '.|     http://docs.smart-mirror.io
      |'._.'|     
-     |     |     To start your mirror:
-   ,'|  |  |`.   > mirror start
+     |     |     To start your mirror you can run both on the pi or via SSH:
+   ,'|  |  |`.   > start-mirror
```

Comment:
You should revert your changes to this file, this will break the new way we want to run the mirror.